dashboard/static/js/dashboard.js

```diff
@@ -0,0 +1,2 @@
+$(document).foundation();
+window.dashboard = {};
```

Comment:
Sort of a nitpick. Since the bundle name is also *dashboard.js*, maybe we could name this *main.js* to reduce the risk of confusion. I called the main stylesheet *main.styl* for the same reason.